chapter_recurrent-neural-networks/rnn-scratch.md

```diff
@@ -160,7 +160,7 @@ def get_params(vocab_size, num_hiddens, device):
 
 ```{.python .input}
 #@tab tensorflow
-def get_params(vocab_size, num_hidden):
+def get_params(vocab_size, num_hidden):  #@save
```

Comment:
If we save it to d2l, let's make the function name more specific to avoid potential future conflict. How about `get_rnn_params`

Reply:
The other frameworks use this same name so we don't want to change those in this PR. We can change them later if we come across any conflicts.

Reply:
In that case, can you apply the name change for all the frameworks? Or I can do it later (just to increase function name readability in the d2l lib)

Reply:
Sure agreed, let's do it in a separate PR so this one can focus on TF.

Reply:
I suggest that we pass in get_params to train_ch8 since the implementation of get_params will be different for e.g. LSTM.
I already created a PR for this change.